defaults/main.yml

```diff
@@ -16,6 +16,16 @@ nginx_multi_accept: "off"
 nginx_error_log: "/var/log/nginx/error.log warn"
 nginx_access_log: "/var/log/nginx/access.log main buffer=16k"
 
+nginx_gzip_enabled: "on"
```

Comment:
Can you rename this var to `nginx_gzip` (without the enabled), as that matches up to the config name and makes it less confusing as to why the value would be on/off when it is 'enabled' (which seems like it would indicate yes/no or true/false).